lib/bap/bap.mli

```diff
@@ -701,7 +701,7 @@ module Std : sig
       (** [string] converts values with the identity function. *)
       val string : string converter
 
-      (** [enum l p] converts values such that unambiguous prefixes of
+      (** [enum l] converts values such that unambiguous prefixes of
           string names in [l] map to the corresponding value of type ['a].
 
           {b Warning.} The type ['a] must be comparable with
```

Comment:
One of the additional bonuses of making `'a converter` abstract is that now we can easily add new field, `compare` that will default to pervasives compare.